src/adapters/web/web.go

```diff
@@ -46,6 +46,11 @@ func (web *Web) Start(port string) error {
 	r.Handle("/images", web.authenticate(web.imagesHandler)).Methods(http.MethodGet)
 	r.Handle("/images", web.authenticate(web.imageUploadHandler)).Methods(http.MethodPost)
 
+	r.Handle("/emails", web.authenticate(web.emailsHandler)).Methods(http.MethodGet)
+	r.Handle("/emails", web.authenticate(web.emailCreateHandler)).Methods(http.MethodPost)
+	r.Handle("/emails/send", web.authenticate(web.emailSendHandler)).Methods(http.MethodPost)
+	r.HandleFunc("/emails/read/{email_id}", web.emailReadHandler).Methods(http.MethodGet)
```

Comment:
this should be `/emails/{id:[0-9]+}/read`

Reply:
Completely forget regular expression!